docs/docs/tutorial/chapter4/authentication.md

```diff
@@ -591,6 +591,14 @@ As you can probably tell by the names:
 * **currentUser**: any details the app has on that user (more on this in a moment)
 * **logOut**: removes the user's session and logs them out
 
+When displaying currentUser, for example {currentUser.id} you may run into a "Cannot read properties of null (reading 'id')" error, this is due to currentUser not being loaded, in this case add  
+```
+if (loading) {
+    return null
+  }
+```
+Also make sure to add loading to variables fetched from useAuth()
```

Comment:
Again, like the above comment - if we can insert this into the BlogLayout codeblock, it makes sense to do so, and it removes the need for this callout. If not, we should probably give more specifics on fetching this variable like showing the snippet (if the following is correct; I have not pulled in loading from useAuth) `const { isAuthenticated, currentUser, logOut, loading } = useAuth()`

Reply:
Sure, will also add the loading as a variable pulled when updating the tutorial.